controllers/storagecluster/storageclasses.go

```diff
@@ -291,7 +291,7 @@ func newCephBlockPoolVirtualizationStorageClassConfiguration(initData *ocsv1.Sto
 	virtualizationStorageClassConfig.storageClass.ObjectMeta.Name = generateNameForCephBlockPoolVirtualizationSC(initData)
 	virtualizationStorageClassConfig.storageClass.ObjectMeta.Annotations["description"] = "Provides RWO and RWX Block volumes suitable for Virtual Machine disks"
 	virtualizationStorageClassConfig.storageClass.Parameters["mounter"] = "rbd"
-	virtualizationStorageClassConfig.storageClass.Parameters["mapOptions"] = "krbd:rxbounce"
+	virtualizationStorageClassConfig.storageClass.Parameters["mapOptions"] = "krbd:rxbounce;krbd:osd_request_timeout"
```

Comment:
osd_request_timeout should not need any timeout value to be set?

Reply:
@nixpanic @idryomov ^^

Reply:
`osd_request_timeout` does need a value: an integer for timeout in seconds.  0 means no timeout.

Reply:
@nixpanic ^^

Reply:
I guess a reasonable value for `osd_request_timeout` might be 120? This should not cause too many temporary interruptions, and a Pod (or VM) would still be able to detect problems in just a few minutes.

Reply:
I am wondering did we add `osd_request_timeout` parameter in the end?

Reply:
@gqlo No, This PR was closed as there was no progress in determining a suitable request timeout number.